internal/dlmanager/store/store.go

```
@@ -113,9 +115,54 @@ func Open(path string) (*Store, error) {
+			continue
+		}
+		if _, err := db.Exec(col.ddl); err != nil {
+			return fmt.Errorf("adding downloads.%s: %w", col.name, err)
+		}
+	}
+	return nil
+}
+
+// hasColumn reports whether a table already defines a column.
+func hasColumn(db *sql.DB, table, column string) (bool, error) {
+	rows, err := db.Query("SELECT name FROM pragma_table_info(?)", table)
```

golangci-lint (2.12.2)

[error] (*database/sql.DB).Query must not be called. use (*database/sql.DB).QueryContext

(noctx)